UnityProject/Assets/Scripts/Objects/Engineering/Reactor/ReactorControlConsole.cs

```diff
@@ -20,6 +55,42 @@ public void SuchControllRodDepth(float requestedDepth)
 			}
 		}
 
+		private void OnDisable()
+		{
+			if (CustomNetworkManager.IsServer == false) return;
+			UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, UpdateMe);
+		}
+
+		private void UpdateMe()
+		{
+			if (ReactorChambers is null) return;
+			if (ReactorChambers.MeltedDown == false || ReactorChambers.PresentNeutrons < 200) return;
```

Comment:
wouldn't this just spam the chat?

Reply:
no, look at the time given again.

Reply:
every eight seconds is still a lot though

Reply:
12 actually

Reply:
though it doesn't matter imo, I think having this scream in your face is a good thing to signify how important it is.

Reply:
especially since when the reactor goes into meltdown, it has like a couple of seconds before it exploads.